plugins/modules/ios_vrf_address_family.py

```diff
@@ -461,15 +461,42 @@
             description: Specify Target VPN Extended Communities.
             type: dict
             suboptions:
-              export:
+              exports:
```

Comment:
Example on how we do attribute deprecation- https://github.com/ansible-collections/cisco.ios/blob/v10.0.0/plugins/modules/ios_bgp_address_family.py#L230